src/components/Select.vue

```diff
@@ -83,6 +83,7 @@
   import pointerScroll from '../mixins/pointerScroll'
   import typeAheadPointer from '../mixins/typeAheadPointer'
   import ajax from '../mixins/ajax'
+  import i18n from '../mixins/i18n'
```

Comment:
I would suggest this, but this is purely subjective as it more or less does the same thing..

```suggestion
   import { ajax, pointerScroll, i18n, pointer as typeAheadPointer  } from '../mixins'
```

## Notice ‼
However, seems you have not added i18n to the exports in [src/mixins/index.js](https://github.com/sagalbot/vue-select/blob/master/src/mixins/index.js ), making my proposal above not working.